### ISSUE TITLE
register command does not behave as expected 

### ISSUE THREAD

**Issue by undertherain** [mbdevpl/ingit#4]
```
ingit register ./
usage: ingit [-h] [--version] [--config PATH] [--repos PATH]
             [predicate] command ...
ingit: error: argument command: invalid choice: './' (choose from 'register', 'clone', 'init', 'fetch', 'checkout', 'merge', 'push', 'gc', 'status')```


``` ingit --version
ingit 0.3.0, Python 3.6.3 (default, Oct 3 2017, 21:45:48) [GCC 7.2.0]
```

**Comment by mbdevpl** [comment]
@undertherain Yeah :) It's fixed in dev version, I'm gonna release it at the end of the week after adding many other features -- for now you could do `pip3.6 install https://github.com/mbdevpl/ingit` (the current version in the repo is `0.3.1.dev17+ff1f1f6a`), `ingit register ./` works as expected in that snapshot, and will work like that in future versions.

**Comment by undertherain** [comment]
it does not crash now, but does not register full path, only starting from the folder where register was called

**Comment by undertherain** [comment]
```
    {
      "name": "Prototype",
      "path": "Projects_heavy/DL/Prototype",
      "remotes": {},
      "tags": []
    }
```

**Comment by undertherain** [comment]
I called `ingit register ./`, looks like it did not expand to absolute path

**Comment by mbdevpl** [comment]
This is fixed in just-released 0.4.8.

I've finally documented how all of this is implemented, so no more surprises I hope.

From current readme:

> The final "path" to the repository stored in the configuration depends on the
> ``repos_path`` in runtime configuation. The configured "path" will be:
> 
> *   resolved absolute path if there is no ``repos_path`` configured or repository path
>     is outside of the ``repos_path``;
> *   resolved relative path to the ``repos_path``, if the repository path is within it;
> *   nothing (i.e. not stored) if the if the repository is stored directly in
>     ``repos_path`` (i.e. there are no intermediate directories).
> 
> Behaviour of storing relative/no paths in some cases is implemented to make
> configuration file much less verbose in typical usage scenarios. To prevent this
> behaviour, and force all repository paths to be absolute, simply set the ``repos_path``
> in your runtime configuraion to JSON ``null``.
